sa.yaml

```diff
@@ -0,0 +1,51 @@
+$schema: http://json-schema.org/draft-07/schema
+title: GA4GH-SA-Definitions
+type: object
+definitions:
+  SequenceFeature:
+    properties:
+      id:
+        type: string
+      location:
+        $ref: "https://raw.githubusercontent.com/ga4gh/vrs/main/schema/vrs.yaml#definitions/Location"
```

Comment:
Seems like we should have a pointer to a more stable URL? Is this a case for putting DOIs over schemas? Not sure if it helps or not ...

Reply:
What's the best way to bring in the VRS location model into the SA spec @larrybabb @ahwagner? Does VRS generate DOIs/IRIs for its location object specs? Like, can we refer to the SequenceLocation object with an IRI?